src/index.ts

```diff
@@ -194,6 +194,35 @@ export class SgidClient {
     }
     return result
   }
+
+  /**
+   * Generates the code verifier (random bytes encoded in url safe base 64) to be used in the OAuth 2.0 PKCE flow
+   * @param length The length of the code verifier to generate (Defaults to 43 if not provided)
+   * @returns The generated code verifier
+   */
+  codeVerifier(length = 43): string {
```

Comment:
nit: Shall we name the functions with a verb prefix, eg. `get...()` or `generate...()` for clearer intentions? I understand the underlying library named them without verb prefixes, but I think the name of the class (`generator`) has conveyed the intention of the function clearly already, which might not be case in our situation.

Reply:
agree with @kschiew. Good to start function name with verb

Reply:
okay, thanks for the comments guys! will make the changes